src/styles/cards.scss

```diff
@@ -0,0 +1,130 @@
+.main {
+  display: flex;
+  justify-content: center;
+}
+
+.main__container {
+  display: flex;
+  align-self: center;
+  flex-wrap: wrap;
+  width: (4 * $card-width) + (3 * $card-margin) + (2 * $inside-padding);
+  padding: 50px 0;
+}
+
+.card {
+  background-color: $white;
+  max-width: $card-width;
+  border: 1px solid $grey;
+  border-radius: 5px;
+  padding: 32px 16px 0;
+  cursor: pointer;
+  margin: 0 24px 46px;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.

Reply:
wasn't fixed 
![image](https://user-images.githubusercontent.com/122989585/224481172-c12eddc8-9869-44fa-b4f0-13d2e4ccc84a.png)